x-pack/plugins/reporting/server/lib/screenshots/get_screenshots.ts

```diff
@@ -21,10 +21,10 @@ const resizeToClipArea = async (
   logger: LevelLogger
 ) => {
   // Check current viewport size
-  const { top, left, width, height } = item.position.boundingClientRect; // the "unscaled" pixel sizes
+  const { width, height, left, top } = item.position.boundingClientRect; // the "unscaled" pixel sizes
   const [viewWidth, viewHeight] = await browser.evaluate(
     {
-      fn: () => [document.body.clientHeight, document.body.clientWidth],
+      fn: () => [document.body.clientWidth, document.body.clientHeight],
```

Comment:
Sometimes it's hard to visually check that we assign height values to height values, and width values to width values. I use a plugin to highlight different strings different colors to give it a color pattern check, and that's what I did here to see that there was a bug.